lib/index.js

```diff
@@ -17,6 +17,25 @@ var shell = require('shelljs');
 var restify = require('restify');
 
 /**
+ * checks if the path is device rooted, that is, if it has [drive letter]:\ at its beggining.
+ * @private
+ * @param {String} fpath
+ *      The file path to check
+ * @returns {Boolean}
+ *      Whether the path is rooted or not.
+ */
+function isWin32DeviceRoot(fpath) {
+    // Only for win32
+    if (process.platform !== 'win32') {
+        return false;
+    }
+
+    // Any drive letter followed by :\
+    var deviceRoot = /^[A-Za-z]:\\/;
+    return deviceRoot.test(fpath);
```

Comment:
`path.isAbsolute` will tell you if root is in the path, and it will work across all platforms.